apps/demo-ui/ui/src/Types.tsx

```diff
@@ -53,7 +53,8 @@ export class SystemChat {
   fromAdhoc: boolean = false;
   editing: boolean = false;
   comment: string = "";
-  filterContent: any;
+  filterContent: FilterValues = {};
```

Comment:
can you make the names consistent: filtersUser or aggregationValues

Reply:
Yes